apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -7,6 +7,22 @@ import {files as filesApi} from '@cdo/apps/clientApi';
 import header from '@cdo/apps/code-studio/header';
```

Comment:
Note: I did some deduplication of the test setup code in this file - viewing without whitespace may be helpful.